src/v0/destinations/redis/transformV2.js

```diff
@@ -0,0 +1,45 @@
+const { transformSubEventTypeProfiles } = require('./transform');
```

Comment:
Do we really need a "v2" file? It doesn't seem like a convention.

Reply:
Fixed it to a utils.js file name